plugins/zendesk/icon_zendesk/actions/update_ticket/action.py

```diff
@@ -17,36 +17,50 @@ def __init__(self):
     def run(self, params={}):
```

Comment:
This run function could definitely use some splitting up to make it more easily tested and maintainable.